config/environments/test.rb

```diff
@@ -36,7 +35,11 @@
 
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
-  PUBLIC_URL = "dev.fsektionen.se"
+  PUBLIC_URL = "example.com"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.